src/Components/ProductList/ProductList.tsx

```diff
@@ -0,0 +1,16 @@
+import ProductCard from '../ProductCard/ProductCard';
+import './ProducrList.css';
+
+const ProductList = () => {
+  return (
+    <div className='ProductListWrapper'>
+        <ProductCard/>
```

Comment:
As I understand, we will use dynamic properties, so one component is enough here

Reply:
Yes, you are right. I will delete extra components when data.js is provided. 
It is duplicated for now just to have a visualization.